src/pages/ExtensionDashboardPage.tsx

```diff
@@ -3,11 +3,28 @@ import WordCountForm from "../components/WordCountForm";
 import WordCountProgressBar from "../components/WordCountProgressBar";
 import { useNavigate } from "react-router-dom";
 import { PROJECT_TITLE_CREATE_NEW, PROJECT_TITLE_NONE } from "../models/projectModels";
-import { CREATE_PROJECT_ROUTE } from "../App";
-import { useState } from "react";
+import { CREATE_PROJECT_ROUTE, SET_GOAL_ROUTE } from "../App";
+import { useEffect, useState } from "react";
+import { UserGoal } from "../models/userModels";
+import { getUserGoal } from "../storage/userGoalStorage";
```

Comment:
I wonder if we should name these something other than `{get/set}UserGoal`. I was super confused by name shadowing since typically in `useState` we’ll name that `setUserGoal` as well